lua/nvim-treesitter/parsers.lua

```diff
@@ -1017,6 +1018,17 @@ list.hyprlang = {
   maintainers = { "@luckasRanarison" },
 }
 
+list.idris = {
+  install_info = {
+    -- TODO: change to
+    -- url = "https://github.com/kayhide/tree-sitter-idris",
+    -- after merge https://github.com/kayhide/tree-sitter-idris/pull/3
```

Comment:
Then we should wait for that. Switching parsers is a breaking change and should be avoided.